packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -708,21 +708,23 @@ class HotRunner extends ResidentRunner {
     const String red = '\u001B[31m';
     const String bold = '\u001B[0;1m';
     const String reset = '\u001B[0m';
-    printStatus(
-      '$fire  To hot reload your app on the fly, press "r". To restart the app entirely, press "R".',
-      ansiAlternative: '$red$fire$bold  To hot reload your app on the fly, '
-                       'press "r". To restart the app entirely, press "R".$reset'
+    if (details)
+      printStatus('Reloading:');
+      printStatus(
+        "$fire  To hot reload your app on the fly, press 'r'; to restart the app entirely, press 'R'.",
```

Comment:
we want this message even if details is false. We found in UX studies that it was critical.